### PR TITLE
S100: David Gao's ultraconnected non-contractible space

### PR DESCRIPTION
**Pull request by prabau**
S100: example of a nonempty ultraconnected non-contractible space.
See https://math.stackexchange.com/questions/5001004.

@david20000813 Please take a look.
You can preview the changes in the `dgao-ultra` branch, by entering that branch in the Advanced tab of pi-base (https://topology.pi-base.org/dev).

I made the name unique enough to identify the space and hint at the main properties.

As for the various traits, you can see which ones were asserted.

In particular, T1 was not asserted but derived, and the same for compact (a kind of non-obvious derivation though).
https://topology.pi-base.org/spaces/S000100/properties/P000016

Among the unknown traits, maybe some of them seem of interest to add: first-countable, second-countable, Alexandrov?

### CODE REVIEW

**Comment by david20000813** [comment]
Thanks! This is quite nice and the derivation for non-compactness is quite surprising. As for other properties, it is second countable since the topology is generated by a countable closed subbasis. It is not Alexandrov - I'll add this to my answer on MSE so that we can just reference that.

**Comment by david20000813** [comment]
Done - the MSE answer now includes a proof that $X$ is second countable as well as a proof that $X$ is not Alexandrov.

**Comment by prabau** [comment]
I don't see your typo fix. Where?

The meager argument is worth adding to the mathse answer and we can just refer to that.
If you want homogeneous to be there too, that's also ok.

**Comment by david20000813** [comment]
@prabau Just added meager to the MSE answer. I’m fine with keeping non-homogeneous just on pi-Base. There’s no reason to add every possible property to the MSE answer.

**Comment by prabau** [comment]
It does not show for me.  Just to go through the procedure for suggesting changes, from the "Files changed" tab, did you click on the `+` to the left of line 11?  And then something pops up to leave a comment.  You can just leave the comment there.  Or optionally, you can also make a "suggestion" by further clicking on the icon for "Add a suggestion".

**Comment by david20000813** [comment]
@prabau That’s a bit weird. I did exactly that, and made a suggestion by clicking on + on line 11. Let me try that again.

File: spaces/S000100/properties/P000086.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+space: S000100
+property: P000086
+value: false
+refs:
+- mathse: 5001004
+  name: Answer to "Are hereditarily connected spaces and ultraconnected spaces contractible?"
+---
+
+The order $\le$ on elements of $X$ is determined by the topology, as it coincides with the specialization order:
+$x<y$ iff $x\subseteq\overline{\{y\}}$.
```
Comment (david20000813) [inline spaces/S000100/properties/P000086.md:11]:
```suggestion
$x\le y$ iff $x\subseteq\overline{\{y\}}$.
```

**Comment by prabau** [comment]
@Almanzoris Would you be able to review this?
(Reminder: wipe out the extended description when doing the "squash and merge")

**Comment by Almanzoris** [comment]
> @Almanzoris Would you be able to review this? (Reminder: wipe out the extended description when doing the "squash and merge")

Sure! I will read it tomorrow.

**Comment by Almanzoris** [comment]
I find it good.

**Review by Almanzoris — APPROVED** [review]

**Comment by Almanzoris** [comment]
@david20000813 I have noticed a small typo in the mathse answer. As it is one character, I can't edit it.

image

**Review by david20000813 — APPROVED** [review]

**Comment by Almanzoris** [comment]
You're welcome! Good.

**Pull request merged**